server.js

```diff
@@ -7,7 +7,9 @@ new WebpackDevServer(webpack(config), {
   contentBase: baseConfig.output.path,
   publicPath: baseConfig.output.publicPath,
   hot: process.env.NODE_ENV === 'development' ? true : false,
-  historyApiFallback: true,
+  historyApiFallback: {
+    disableDotRule: true,
+  },
```

Comment:
I'm thinking the "true" was only there by accident. But I see your point, let's not hold up this PR because of it.